src/test/resources/com/puppycrawl/tools/checkstyle/asttreestringprinter/InputAstTreeStringPrinterFullOfSinglelineComments.java

```diff
@@ -61,4 +61,4 @@
 }//60
 }//61
 }//62
-//63
\ No newline at end of file
+//63
```

Comment:
I recommend to move all such supplemental changes to separate PR with commit with messge "minor: ....", I will merge it instantly, so this PR will be less in size and more fcused in on config.

Reply:
If we really want to use the plugin, which kind of makes sense at least in the beginning, I would recommend splitting the introduction as it's done here: [checkstyle/checkstyle#16598](https://github.com/checkstyle/checkstyle/pull/16598). Then, build from there by removing the exclusions as far as possible and useful. This way, the PRs are really focused on one concern only.

1. Add the config: [checkstyle/checkstyle#16574](https://github.com/checkstyle/checkstyle/pull/16574)
2. Add the plugin: [checkstyle/checkstyle#16598](https://github.com/checkstyle/checkstyle/pull/16598)
3. Apply the plugin: [checkstyle/checkstyle#16604](https://github.com/checkstyle/checkstyle/pull/16604)